samples/snippets/csharp/concepts/linq/LinqSamples/GroupByContiguousKeys.cs

```diff
@@ -56,36 +54,35 @@ class Chunk<TKey, TSource> : IGrouping<TKey, TSource>
     // has a reference to the next ChunkItem in the list.
     class ChunkItem
     {
-        public ChunkItem(TSource value)
-        {
-            Value = value;
-        }
+        public ChunkItem(TSource value) => Value = value;
         public readonly TSource Value;
-        public ChunkItem? Next = null;
+        public ChunkItem? Next;
     }
 
+    public TKey Key { get; }
+
     // Stores a reference to the enumerator for the source sequence
-    private IEnumerator<TSource> enumerator;
+    IEnumerator<TSource> enumerator;
```

Comment:
revert this change. Our style guide recommends adding the access modifier, even when it's the default value (except when explaining the default access modifiers):

```suggestion
    private IEnumerator<TSource> enumerator;
```